cpp/src/arrow/compute/kernels/codegen_internal.h

```diff
@@ -445,6 +445,29 @@ static void VisitTwoArrayValuesInline(const ArrayData& arr0, const ArrayData& ar
                         arr0.length, std::move(visit_valid), std::move(visit_null));
 }
 
+template <typename Arg0Type, typename Arg1Type, typename Arg2Type, typename VisitFunc,
+          typename NullFunc>
+static void VisitThreeArrayValuesInline(const ArrayData& arr0, const ArrayData& arr1,
```

Comment:
This doesn't seem used at all in this PR, which is it here?

Reply:
Removed.